.codecov.yml

```diff
@@ -0,0 +1,12 @@
+codecov:
+  branch: dev
+coverage:
+  status:
+    project:
+      default:
+        target: 90
+        threshold: 0.09
+  notify:
+    slack:
```

Comment:
slack?

Reply:
It's hooked up to a Discord webhook ending in `/slack` going to #codecov

Reply:
let's add a comment